### DISCUSSION TITLE
Group Token

### DISCUSSION THREAD

**Discussion by CaptainBrando283** [Muttley/foundryvtt-shadowdark#801] [Ideas]
Hi
I do not know if this is the right place for this, looking through the code (trying to learn) I do not see a group token like the DND5e system has. Hex crawls are apart of ShadowDark Rpg, could you add a group actor? Please feel free to delete this and correct me if that is already a feature and I am just am not seeing it.

**Comment by Brittonlj** [comment]
For what it is worth, I just create an NPC for the group and give everyone ownership of it, so they can see through it and control it.
